.tekton/multicluster-operators-application-acm-211-pull-request.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/stolostron/multicloud-operators-application?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/cancel-in-progress: "true"
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
+      == "release-2.11"
+  creationTimestamp: null
+  labels:
+    appstudio.openshift.io/application: release-acm-212
```

Comment:
@mikeshng  Since this is for release-2.11 branch, all 212 should be replaced with 211